ui/src/Header/Header.test.tsx

```diff
@@ -17,24 +17,34 @@
 
 import React from 'react';
 import {act, fireEvent, RenderResult, wait} from '@testing-library/react';
+import { axe, toHaveNoViolations } from 'jest-axe';
 import Header from './Header';
 import TestUtils, {renderWithRedux} from '../tests/TestUtils';
 import {PreloadedState} from 'redux';
 import {GlobalStateProps} from '../Redux/Reducers';
 import {RunConfig} from '../index';
 
-jest.useFakeTimers();
 const debounceTimeToWait = 100;
+expect.extend(toHaveNoViolations);
 
 describe('Header', () => {
+    const expectedUrl = 'https://some-url';
     const initialState: PreloadedState<GlobalStateProps> = {currentSpace: TestUtils.space} as GlobalStateProps;
     let app: RenderResult;
 
     beforeEach( async () => {
+        window.location = {href: expectedUrl} as Location;
```

Comment:
Upon talking w/ Ashley, we've determined that this isn't being used in the tests in this file, so we're making an incremental commit removing it to reduce noise in our test file.